css/index.css

```
@@ -1,6 +1,20 @@
-
+/*banner__text의 br 역할을 수행 해준다.*/
 .text-wrap{
     display: block;
+    font-size: 100%;
+}
+
+/*접근성을 고려해 sr-only 기법 사용 */
+.sr-only{
+    position: absolute;
+	width: 1px;
+	height: 1px;
+	padding: 0;
+	margin: -1px;
+	overflow: hidden;
+	clip-path: inset(50%);
+	border: 0;
+	clip: rect(0 0 0 0);
```

💊 제안
현재 코드에서는 속성별 들여쓰기가 달라 가독성이 조금 떨어질 수 있습니다.  
계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
    position: absolute;
    width: 1px;
    height: 1px;
    padding: 0;
    margin: -1px;
    overflow: hidden;
    clip-path: inset(50%);
    border: 0;
    clip: rect(0 0 0 0);
```